src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -265,7 +267,7 @@ private void apply(RoutingContext routingContext) {
           .mapTo(Location::from)
           .whenNotFound(failed(new ServerErrorFailure("Can`t find location")))
           .fetch(request.params().get(LOCATION_ID_NAME))
-          .thenCompose(r -> r.after(location -> getPolicyId(request.params(), drools, location)))
+          .thenCompose(r -> r.after(location -> getPolicyIdAndConditionsList(request.params(), drools, location)))
```

Comment:
Is there a more succinct name for this, could this be a policy or rule match or something similar?

Reply:
renamed